lighthouse-core/lib/dependency-graph/simulator/tcp-connection.js

```diff
@@ -89,11 +89,15 @@ class TcpConnection {
    *  https://hpbn.co/transport-layer-security-tls/#tls-handshake for details.
    *
    * @param {number} bytesToDownload
-   * @param {number=} timeAlreadyElapsed
-   * @param {number=} maximumTimeToElapse
+   * @param {{timeAlreadyElapsed: number, maximumTimeToElapse}=} options
    * @return {{timeElapsed: number, roundTrips: number, bytesDownloaded: number, congestionWindow: number}}
    */
-  simulateDownloadUntil(bytesToDownload, timeAlreadyElapsed = 0, maximumTimeToElapse = Infinity) {
+  simulateDownloadUntil(bytesToDownload, options) {
+    options = Object.assign({
```

Comment:
can we destructure AND do default values?

based on http://node.green/ i think we're good.

Reply:
totes